azure-pipelines/templates/update-homebrew-formula.yml

```diff
@@ -0,0 +1,47 @@
+  
+jobs:
+  - job: Update_Homebrew_Formula
+    timeoutInMinutes: 30 # timeout on job if deploy is not completed in 30 minutes
+    pool:
+      vmImage: ubuntu-16.04
+    steps:
+      - download: current
+
+      - task: DownloadPipelineArtifact@2
+        inputs:
+          artifact: 'BedrockPackage'
+          path: $(Agent.BuildDirectory)
+
+      - checkout: BedrockHomebrewCore
+        persistCredentials: true
+        clean: true
+        
+      - script: |
+          # Read version and sha256 number
+          sha_number=$(sed -n '2p' < $(Agent.BuildDirectory)/BedrockPackage/package-info.txt)
+          version_tag=$(sed -n '1p' < $(Agent.BuildDirectory)/BedrockPackage/package-info.txt)
+          echo "version tag"
+          echo $version_tag
+
+          dir $(Build.SourcesDirectory)
+          cd $(Build.SourcesDirectory)
+
+          # Add changes to the homebrew formula repository
+          git pull origin master
+          git checkout -b master
+
+          # Set git identity
+          git config user.email "admin@azuredevops.com"
+          git config user.name "Automated Account"
+
+          cd Formula
```

Comment:
@edaena we may want to add a comment saying that we are parsing the file to find the version, update it and commit and push